frontend/store/companies/actions.js

```diff
@@ -0,0 +1,66 @@
+import types from "./mutation-types";
+
+export default {
+  async setCompanies({ commit }) {
+    await fetch("/api/faculties/wydzial-techniczny/companies.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_COMPANIES, data);
+      });
+  },
+  async setFilters({ commit }) {
+    await fetch("/api/faculties/wydzial-techniczny/filters.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_FILTERS, data);
+      });
+  },
+  setFaculties({ commit }) {
+    fetch("/api/faculties/faculties.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_FACULTIES, data);
+      });
+  },
+  setDocuments({ commit }) {
+    fetch("/api/faculties/wydzial-techniczny/documents/documents.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_DOCUMENTS, data);
+      });
+  },
+  setTags({ commit }) {
+    fetch("/api/faculties/wydzial-techniczny/filters.json")
```

Comment:
You forgot await I believe.

Reply:
These don't need to by asynchronous, but I can add it.

Reply:
Oh, I'm a stupid. Of course you're using `.then` here, so `await` is redundant. But still, just make generic function for that I mentioned above.

Reply:
Okay, it's already fixed.